lark/grammars/python.lark

```diff
@@ -190,7 +194,7 @@ arguments: argvalue ("," argvalue)*  ("," [ starargs | kwargs])?
 
 starargs: stararg ("," stararg)* ("," argvalue)* ["," kwargs]
 stararg: "*" test
```

Comment:
```suggestion
stararg: "*" test | assign_expr -> assign_expr
```

I'm not sure about this one, but that's roughly how it is in the [docs](https://docs.python.org/3/reference/expressions.html#calls)